lib/profile/sections/contact.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:flutter/material.dart';
+import 'package:mentorApp/constants/profile_constants.dart';
+import './header.dart';
+ 
+class ProfileContact extends StatelessWidget {  
+  final Map contact;
+  ProfileContact({Key key, @required this.contact}) : super(key: key);
+  
+  final IconData email = const IconData(0xf422,
+          fontFamily: CupertinoIcons.iconFont,
+          fontPackage: CupertinoIcons.iconFontPackage);
+
+  final IconData phone = const IconData(0xf4b8,
+          fontFamily: CupertinoIcons.iconFont,
+          fontPackage: CupertinoIcons.iconFontPackage);
+
+  final IconData web = const IconData(0xf4d2,
+          fontFamily: CupertinoIcons.iconFont,
+          fontPackage: CupertinoIcons.iconFontPackage);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: MediaQuery.of(context).size.width,
+      padding: EdgeInsets.fromLTRB(30.0, 10.0, 30.0, 20.0),
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            width: 5.0,
+            color: Colors.black12
+          )
+        )
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          SectionHeader(title: SECTION_TITLE_CONTACT),
+          buildInfoRow(
```

Comment:
same as above, can you generate these InfoRows by looping or mapping?

Reply:
Same reply to your comment above! In this case, I'd have to map icons, labels, infos, and handlers to the `buildInfoRow` widget. Probably doable if I create an list of objects containing those but I feel like that's counter-productive